src/sentry/api/helpers/group_index/update.py

```diff
@@ -738,3 +724,32 @@ def update_groups(
         result["inbox"] = inbox
 
     return Response(result)
+
+
+def handle_has_seen(
+    has_seen: Any,
```

Comment:
What might `has_seen` be other than a boolean?

Reply:
Huh. Is that a test problem, though, or a real problem? (You could cast it to a boolean as you're feeding it to the function, I suppose.)

IMHO it's helpful to be as specific as we can with types, but totally up to you if you want to do that as part of this PR or not.